.github/workflows/stable-release.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    branches:
+      - 'stable'
```

Comment:
We don't need the branch `stable`. There will only be a file somewhere named `stable`.